apps/monarch/monarch-frontend/src/app/actionsController.ts

```diff
@@ -136,53 +137,63 @@ async function fetchPendingPractitioners(useFake = false, accessToken: string):
 export function makeActionsController(): ActionsController {
   // Mutable in-memory store of therapists
   // [] represents that the store has not been initialized yet
-  let allPractitioners: Therapist[] = [];
+  let therapists: Therapist[] = [];
```

Comment:
Cleaning up the logic here to only do filtering on a condition if that condition is given - previous code was a little confusing